static/locales/en-US.yaml

```diff
@@ -1,5 +1,5 @@
 # Put the name of your locale in the same language
-Locale Name: English (US)
+Locale name: English (US)
```

Comment:
https://github.com/search?q=repo%3AFreeTubeApp%2FFreeTube+%22Locale+Name%22+language%3AJavaScript&type=code

This key is used but I see no point updating it

Reviewing other changes later

Reply:
Makes sense. There are more changes in here like that. I would like to hear from the others first before i start changing stuff though